### ISSUE TITLE
Rocky Linux

### ISSUE THREAD

**Issue by gizmoguy** [fsmg/mirror-requests#20]
Website: https://rockylinux.org
Details on becoming a mirror:

> If you’re interested in mirroring Rocky Linux, we’ve set up Mirror Manager, please email infrastructure@rockylinux.org or drop a line in the ~Infrastructure channel on Mattermost..

**Comment by gunkaaa** [comment]
Yes please! Rocky and Almalinux are apparently the successors to CentOS.

There's a detailed mirroring guide at https://docs.rockylinux.org/guides/mirror_management/add_mirror_manager/
